src/include/storage/index/index_builder.h

```diff
@@ -67,7 +72,39 @@ class IndexBuilder {
     return *this;
   }
 
+ /**
+  * @param sql_table, transaction context
+  * @return the builder object
+  */
+  IndexBuilder &SetSqlTableAndTransactionContext(common::ManagedPointer<storage::SqlTable> sql_table,
+                                                 const common::ManagedPointer<transaction::TransactionContext> txn) {
+    TERRIER_ASSERT(sql_table == nullptr && txn == nullptr || sql_table != nullptr && txn != nullptr,
+                   "sql_table / txn is null and txn / sql_table is not.");
```

Comment:
better phrased as:   sqltable and txn are both null, or sqltable and txn are both not null

Reply:
the phrasing still seems logically incorrect